### ISSUE TITLE
rename MTG to something less confusing

### ISSUE THREAD

**Issue by FatalError42O** [minetest/minetest_game#3068]
## Problem

Tons of people are introduced to Minetest on regular basis- most of the time they're confused by the difference between Minetest Game and Minetest, mainly because (and this is a real shocker) it's a bad name. 

Newcomers think that this is all there is to Minetest, but MTG has essentially zero content- it's not even *a game* let alone what should represent Minetest. It's bad enough that it ships with Minetest, but it certainly shouldn't be called what it is now. I'm tired of having to explain to people that there's a difference between "Minetest" and "Minetest Game"! And I'm sure we've lost hundreds of potential players who just think this is a lame Minecraft clone and not a modding community.

## Potential names
here's a few potential name ideas I threw together (really fast) that'd be probably better then what we have now for the sake of clarity:

- MeseTest Game
- MeseTest
- Mese Game
- Basic Game
- Basic (yes just "basic", because that'd describe what it is, at least)
- Minetest Base 
- BaseTest
- MTG (better then having Minetest explicitly in the name)

if you have (likely much better) suggestions, leave it in the comment of this issue/discussion and I'll add it to the list

## Alternatives

even if we don't rename it, something has to be done for clarity. Maybe a message when you make a new world with MTG that tells you "there's more to Minetest, Minetest Game is just a template" or something like that. There has to be a better way then this. Or maybe even just stop shipping MTG with minetest, seeing as it is a TEMPLATE for a game more then anything.

**Comment by GNUAn** [comment]
Basetest is good name

**Comment by appgurueu** [comment]
1. Since we are probably renaming Minetest eventually, this is most probably low-priority.
2. MTG is in feature freeze, so a rename makes particularly little sense. It would just create unnecessary confusion: Usually a rename means you plan significant work or signifies a change in direction, neither of which is the case here.
3. MTG has been debundled from MTE.
4. This issue is controversial (look at the reactions).

In conclusion, a rename of MTG is not worth the hassle at all. Closing.
